go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/go-debos/fakemachine
+
+go 1.11
+
+require (
+	github.com/docker/go-units v0.4.0
+	github.com/jessevdk/go-flags v1.5.0
+	github.com/stretchr/testify v1.7.0
+	github.com/surma/gocpio v1.0.2-0.20130630172628-c01aebd00e02
```

Comment:
you also need to add another step in the commit MR to run `go mod tidy` after editing the tag :-)

Reply:
it's already there:
```
1. go mod init github.com/go-debos/fakemachine
2. go mod tidy
3. edit go.mod as outlined below
4. re-run, go mod tidy
```